docs/en/latest/getting-started.md

```diff
@@ -19,7 +19,41 @@ title: Getting Started
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
--->
+-->  
+
+## What is an API Gateway?
```

Comment:
Maybe we can delete the question mark here? Since the other titles are texts only.
```suggestion
## What is an API Gateway
```